beacon-chain/sync/subscriber.go

```diff
@@ -122,6 +122,18 @@ func (r *RegularSync) subscribe(topic string, validate validator, handle subHand
 			return
 		}
 	}
+	go func() {
```

Comment:
the placement isnt correct, this will spin off a chainstart watcher goroutine for each topic subscription, we only need to do it once

Reply:
thanks, testing it now